2015/intro_concurrency.slide

```diff
@@ -82,7 +82,7 @@ Go is easy (language design):
 - Simple syntax (discussed later)
 - Garbage collected (no manual memory management)
 - Stack traces on nil dereference (not "segmentation fault")
-- Pointers but automatically on stack wherever possible
+- Pointers put automatically on stack wherever possible
```

Comment:
I think I meant the "but" originally, but it's a bad sentence. What really should be said here is that there are pointers, but you don't need to worry about them being heap allocated or stack allocated (they can't disappear on you).